package.json

```diff
@@ -1,11 +1,14 @@
 {
   "private": true,
   "scripts": {
-    "build": "./scripts/generate.js"
+    "build": "./scripts/generate.js",
+    "//": "lint command ignores old files",
+    "lint": "markdownlint-cli2 \"**/*.md\" \"#node_modules\" \"#201[0-9]/*.md\" \"#202[0-2]/*.md\" \"#2023/01.md\""
   },
   "devDependencies": {
     "dedent": "^0.7.0",
     "glob": "^7.2.3",
-    "markdown-magic": "^0.1.25"
+    "markdown-magic": "^0.1.25",
+    "markdownlint-cli2": "^0.7.1"
```

Comment:
btw why cli2 instead of cli? i use cli on another repo

Reply:
probably nothing wrong with `cli` but `cli2` is from the same maintainer as `markdownlint`